lib/sidekiq_alive/redis/base.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module SidekiqAlive
+  module Redis
+    class Base
+      def set(...)
+        raise(NotImplementedError)
+      end
```

Comment:
```suggestion
      def set(*)
        raise(NotImplementedError)
      end
```

Use `*` if you don't care about the arguments being passed. `...` assumes they're forwarded.